cif_pow.dic

```diff
@@ -12689,4 +12689,7 @@ save_
 
        Converted all 'Array' and 'Matrix'-type data items to be 'List', except
        _pd_pref_orient_march_dollase.hkl.
+
+       Renamed _pd_calc.component_intensity_*_list to
+       _pd_calc.component_intensities_*
```

Comment:
I think this message can be omitted if the `_pd_calc.component_intensity_*_list` and `_pd_calc.component_intensities_*` items were introduced in the same version of the dictionary in which they were renamed (that is, in version 2.5.0).

Reply:
Can do.